### ISSUE TITLE
[BUG] New javascript does not work for multivalue fields

### ISSUE THREAD

**Issue by derhansen** [einpraegsam/powermail_cond#77]
Thanks for the vanilla JS refactoring - it is highly appreciated 👍

There is however a problem with multivalue fields (e.g checkbox fields). When a checkbox field can have multiple values, then the `<f:form.checkbox>` viewhelper will render a hidden input field like `<input type="hidden" name="tx_powermail_pi1[field][myfield]" value="">`.

The `#getFieldByMarker` JavaScript function will however only query for non-hidden fields ([code](https://github.com/einpraegsam/powermail_cond/blob/develop/Resources/Private/Build/JavaScript/PowermailConditions.js#L150)), so it will return a null value for fields with multiple values. 

Further hide/show field actions are then stopped due to a JavaScript error:

```
PowermailCondition.js?1678367622:58 TypeError: Cannot read properties of null (reading 'closest')
    at #hideField (PowermailCondition.js?1678367622:124:35)
    at #processActions (PowermailCondition.js?1678367622:78:30)
    at PowermailCondition.js?1678367622:54:31
```

**Comment by einpraegsam** [comment]
Hi Torben,

I just digged into that issue and faced also another one: It's even not possible hide or show submit buttons because of a similar problem. I just tried to fix this issue with a commit in develop.
Can you please test if this would help you?

Thx, Alex

**Comment by derhansen** [comment]
@einpraegsam Just checked out the development branch and this does not fully fix the problem. When you have a field (e.g. Multivalue checkbox field, which is mandatory depending on a condition), the current JavaScript does not work. I the mandatory field is hidden based on a condition, it is not possible to submit the form, since the validation still requires the hidden field to be filled out.

It must therefore be considered, that hidden mandatory multivalue fields get de-required or re-required (see my PR #78)

**Comment by einpraegsam** [comment]
Thx @derhansen back to work :)

**Comment by einpraegsam** [comment]
I pushed another fix to develop branch. @derhansen now the problem with mandatory checkboxes as target field should be fixed.
@fazzyx I'm not sure if your problem is still solved or if it appears again now. Can you please re-test?

**Comment by einpraegsam** [comment]
@derhansen So, I just checked with all kind of fieldtypes and everything should work as expected. If not, I will sponsor a glass beer to you :)
Can you please try again?

**Comment by derhansen** [comment]
@einpraegsam Thanks a lot for your work on this issue and on the extension. All my tests are green now, so the issue is resolved 👍

**Comment by einpraegsam** [comment]
Thx, I just released 10.1.0 to packagist

**Comment by fazzyx** [comment]
> @fazzyx I'm not sure if your problem is still solved or if it appears again now. Can you please re-test?

Yes, it is working. Many thanks for your work!
